src/main/java/Configuration/JettyConfiguration.java

```diff
@@ -1,16 +1,12 @@
-package me.arminb;
+package Configuration;
 
+import me.arminb.sara.JettyServer;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-
-@Configuration
-public class MyConfiguration {
 
+public class JettyConfiguration {
     @Bean
     public JettyServer jettyServer() {
         JettyServer server = new JettyServer();
-        //server.setName("your foo");
         return server;
```

Comment:
This is a good place to pass port number in Jetty Server constructor. Later you can have a another bean injected here from which you can read the port number. That bean would be responsible for reading the configuration from a .properties or .yml file.